data/en/mbs_0251.json

```diff
@@ -0,0 +1,735 @@
+                                    ]
+                                },
+                                "type": "TextArea",
+                                "mandatory": true,
+                                "label": "Comments",
+                                "id": "changes-in-turnover-answer-3",
+                                "q_code": "146h"
+                            }],
+                            "description": "<p>We rely on your commentary to ‘tell the story’ behind changes in figures.  By commenting here it will reduce the need for us to call you.</p>"
+                        }]
+                    },
+                    {
+                        "id": "number-of-employees-total-block",
+                        "questions": [{
+                            "answers": [{
+                                "id": "number-of-employees-total",
+                                "label": "Total number of employees",
+                                "mandatory": false,
+                                "q_code": "50",
+                                "type": "Number",
+                                "default": 0
+                            }],
+                            "description": "An employee is anyone aged 16 years or over that your organisation directly pays from its payroll(s), in return for carrying out a full-time or part-time job or being on a training scheme.",
+                            "guidance": {
+                                "content": [{
+                                        "title": "Include",
+                                        "list": [
+                                            "all workers paid directly from this business’s payroll(s)",
+                                            "those temporarily absent but still being paid, for example on maternity leave"
+                                        ]
+                                    },
+                                    {
+                                        "title": "Exclude:",
+                                        "list": [
+                                            "working owners who are not paid via PAYE",
+                                            "agency workers paid directly from the agency payroll",
+                                            "voluntary workers",
+                                            "former employees only receiving a pension",
+                                            "self-employed workers"
+                                        ]
+                                    }
+                                ]
+                            },
+                            "id": "number-of-employees-total-question",
+                            "title": "On {{exercise.employment_date|format_date}}, what was the number of employees for {{respondent.trad_as_or_ru_name}}?",
+                            "type": "General"
+                        }],
+                        "routing_rules": [{
```

Comment:
You don't need this routing rule as is breaking it for me as I can't get past the employment question with a blank/zero. The skip condition and routing rule on ConfirmationQuestion should deal with this.